src/components/PieChart/Tooltip.tsx

```diff
@@ -38,25 +42,25 @@ const Tooltip = ({ data }: TooltipProps) => (
   <div tabIndex={-1} role="tooltip">
     <StyledList>
       <StyledItem>
-        <Text as="p" variant="body" prominence="stronger">
+        <StyledText as="p" variant="body" prominence="stronger">
```

Comment:
Could we instead have a flex space-between on the StyledItem ? 

We could remove Space also